DataFormats/SiStripCluster/interface/SiStripApproximateClusterv1.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+#define DATAFORMATS_SISTRIPAPPROXIMATECLUSTERv1_H
+
+#include <numeric>
+#include "DataFormats/SiStripCluster/interface/SiStripCluster.h"
+
+class SiStripApproximateClusterv1  {
+public:
+  
+  SiStripApproximateClusterv1() {}
+  explicit SiStripApproximateClusterv1(const SiStripCluster& cluster){
+    barycenter_=static_cast<uint16_t>(cluster.barycenter());
+    width_=cluster.size();
+    avgCharge_ = static_cast<uint8_t>(cluster.charge()/cluster.size());
```

Comment:
in case width>width max, should this be renormalized so that total charge is correct?
Anyhow not sure what one can do with a compact cluster with width>128

Reply:
btw always protect from overflow: uint will wrap

Reply:
Not sure on how to renormalize. We are indeed stripping off anyway a fraction of the charge of big clusters. What we can do is to yes, rescale up to saturation but also that would be an approximation. If you think it may be useful, I'll implement it.

Same discussion is for approximation. 

Long clusters (>64) are quite often present in HI because sometimes you get a loopers and another signal particle on top.